lib/procinfo_unix.cpp

```diff
@@ -263,3 +263,24 @@ int procinfo_setup(PROC_MAP& pm) {
     find_children(pm);
     return 0;
 }
+
+// get total user-mode CPU time
+// see https://www.baeldung.com/linux/get-cpu-usage
+//
+double total_cpu_time() {
+    char buf[1024];
+    static FILE *f=NULL;
+    static double scale;
+    if (!f) {
+        f = fopen("/proc/stat", "r");
+        long hz = sysconf(_SC_CLK_TCK);
+        scale = 1./hz;
+    } else {
+        fflush(f);
```

Comment:
I believe `fflush()` is unnecessary if followed by `rewind()`  (which is basically `fseek()`).

Reply:
without fflush(), fgets() will return the same buffered data

Reply:
FWIW I am not totally convinced that this is the case, since fseek() will also flush the data (and rewind() is based on fseek()). At least if my memory serves me right.